src/image.tsx

```diff
@@ -52,15 +69,15 @@ export type ImageProps = {
 };
 
 const Image = React.forwardRef<HTMLImageElement, ImageProps>(
-    ({aspectRatio = '1:1', alt = '', dataAttributes, noBorderRadius, ...props}, ref) => {
+    ({aspectRatio = '1:1', alt = '', dataAttributes, noBorderRadius, src, ...props}, ref) => {
+        const supportsAspectRatio = useSupportsAspectRatio();
         const ratio = typeof aspectRatio === 'number' ? aspectRatio : RATIO[aspectRatio];
         const noBorderRadiusContext = useDisableBorderRadius();
         const noBorderSetting = noBorderRadius ?? noBorderRadiusContext;
         const classes = useStyles({
             noBorderRadius: noBorderSetting,
             aspectRatio: !props.width && !props.height ? ratio : undefined,
```

Comment:
I think I don't understand this condition :/
shouldn't we always pass the `aspectRatio`, @pladaria ?

Reply:
fixed the logic